pkg/state/state.go

```diff
@@ -3192,6 +3192,15 @@ func renderValsSecrets(e vals.Evaluator, input ...string) ([]string, error) {
 	return output, nil
 }
 
+func hideChartCredentials(chartCredentials string) string {
+	httpIndex := strings.Index(chartCredentials, "http")
```

Comment:
why did only have http? thanks. @NavesEdu

Reply:
Only the charts that were passed the links through the helmfile presented this credentials exposure problem. So I throught it would be better to hide everything from "http" to "@".

Reply:
The result looks a bit strange. http://****@example.com or https://***@example will be better. @NavesEdu

Reply:
Fixed @yxxhero, thank you!

Reply:
The result looks a bit strange. sorry for the error display.
```
http://***@example.com or https://***@example will be better. 
```

@NavesEdu

Reply:
No more. Please add more test cases. @NavesEdu